core/src/main/java/com/sequenceiq/cloudbreak/service/ClusterCommonService.java

```diff
@@ -208,6 +215,34 @@ public void setMaintenanceMode(Stack stack, MaintenanceModeStatus maintenanceMod
         }
     }
 
+    /**
+     * Get cluster host details (ips + cluster name) - ini format
+     * @param cluster cluster object that is used to fill the cluster details ini
+     * @return Ini file content in string
+     */
+    public String getHostNamesAsIniString(Cluster cluster) {
+        Long clusterId = cluster.getId();
+        String clusterName = cluster.getName();
+        String serverHost = cluster.getAmbariIp();
+        Set<HostMetadata> agentHostsSet = hostMetadataService.findHostsInCluster(clusterId);
+        if (agentHostsSet.isEmpty()) {
+            throw new NotFoundException(String.format("Not found any agent hosts (yet) for cluster '%s'", cluster.getId()));
+        }
+        String agentHosts = agentHostsSet.stream()
+                .map(HostMetadata::getHostName)
+                .collect(Collectors.joining("\n"));
+        return String.join(
+                "\n",
+                addSectionWithBody("cluster", "name=" + clusterName),
+                addSectionWithBody("server", serverHost),
```

Comment:
I have just realized that, wouldn't it be better to make inventory host group based?

[master]
...

[worker]
...

[compute]
....

Reply:
@akanto , ok, i can update that, one of the UT is failed anyway (RestUrlParserTest ... because of the new endpoint)
probably will keep the server, but with renaming to gateway then, also i can still include the agents as well next to the hostgroups (in order to use commands on all hosts)